Assets/Scripts/CalorieManager.cs

```diff
@@ -0,0 +1,187 @@
+﻿using Photon.Pun;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class CalorieManager : MonoBehaviourPunCallbacks {
+    public Canvas calorieCanvas;
+    public GameObject 
+        genderPanel,
+        agePanel,
+        weightPanel,
+        caloriePanel;
+    public Text caloriesBurntText;
+
+    private bool isMovesenseConncted = false;
+    private float matchTime = 0.0f;
+    private float timeElapsed;
+
+    private List<int?> heartRateBuffer = new List<int?>();
+    private List<int?> aveHeartRates = new List<int?>();
+    private bool isCalculating = false;
+
+    #if UNITY_EDITOR
+    private Camera playerCam;
+    #endif
+
+    [Header("Desktop")]
+    [SerializeField]
+    private GameObject[] desktopObjects = null;
+
+    [Header("OVR")]
+    [SerializeField]
+    private GameObject[] oculusObjects = null;
+    private LineRenderer laserLineRenderer = null;
+
+    void Start() {
+        if (OVRPlugin.productName != null && OVRPlugin.productName.StartsWith("Oculus")) {
+            foreach (GameObject go in desktopObjects) {
+                go.SetActive(false);
+            }
+            foreach (GameObject go in oculusObjects) {
+                go.SetActive(true);
+
+                // UIHelpers must be the first go in oculusObjects
+                if (laserLineRenderer == null) {
+                    laserLineRenderer = go.GetComponentInChildren<LineRenderer>();
+                }
+            }
+        }
+
+        #if UNITY_EDITOR
+        Camera[] cams = FindObjectsOfType<Camera>();
+        foreach (Camera c in cams) {
+            PhotonView pv = c.GetComponentInParent<PhotonView>();
+            if (pv == null || !c.isActiveAndEnabled) {
+                continue;
+            } else {
+                playerCam = c;
+                break;
+            }
+        }
+        calorieCanvas.renderMode = RenderMode.ScreenSpaceOverlay;
+        calorieCanvas.worldCamera = playerCam;
+        GetComponent<VRFollowCanvas>().enabled = false;
+        #endif
+
+        calorieCanvas.gameObject.SetActive(false);
+        GameManager.Instance.TimeOverEvent.AddListener(ShowCalorieUI);
+        // If player connected a movesense device
+        if (MovesenseSubscriber.instance != null && MovesenseSubscriber.instance.heartRate != null) {
+            Debug.Log("CalorieManager: Movesense is CONNECTED, recording heartrate");
+            isMovesenseConncted = true;
+            isCalculating = false;
```

Comment:
this line is either redundant (since set to false from field initialisation) or should be moved out of this `if` (so that it applies to those without movesense connected too)

Reply:
Its only required if movesense is connected though.